lib/models/votes-aggregations.js

```diff
@@ -0,0 +1,19 @@
+// count the number of total votes on a poll and the number of votes for each option
+const voteAmount = [
+  {
+    '$project': {
+      'option': '$option'
```

Comment:
This should be 'option': '$options' (with and s!)
Your Vote model calls that field options.